packages/atomic/.storybook/main.mts

```diff
@@ -34,77 +34,21 @@ function externalizeDependencies() {
     name: 'externalize-dependencies',
     enforce: 'pre',
     resolveId: (id: string) => {
-      if (id.startsWith('/headless')) {
+      if (/^\/(headless|bueno)/.test(id)) {
         return false;
       }
-      if (packageMappings[id]) {
+      if (generateExternalPackageMappings(__dirname)[id]) {
         if (!isCDN) {
```

Comment:
Isn't this superfluous? We're already checking `isCDN` here: https://github.com/coveo/ui-kit/pull/4433/files#diff-5dda87fd1db013e00675717666c12d3ee6a53108c325e257ead47ef5b9573d0cR27 and this seems to be the only place where we're calling `generateExternalPackageMappings`, so this condition should never be satisfied.